Braintree/src/main/java/com/braintreepayments/api/Venmo.java

```diff
@@ -48,14 +48,15 @@
     static final String CERTIFICATE_ISSUER = "CN=Andrew Kortina,OU=Engineering,O=Venmo,L=Philadelphia,ST=PA,C=US";
```

Comment:
Some of these static constants could probably be removed if we want to do some `v3` style due diligence.

Reply:
Good call yeah removed those and some old tests